vscode-client/snippets/snippets.json

```diff
@@ -0,0 +1,214 @@
+{
+    "shebang": {
+        "description": "[sh]hebang",
+        "prefix": "sh",
```

Comment:
I'm unsure about the convention here, but at least for JavaScript/TypeScript project the snippets usually have a more descriptive name/prefix. I would personally prefer `shebang` compared to `sh`, `while` compared to `wh`, `for` instead of `fo`, etc. 

<img width="812" alt="Screenshot 2023-01-09 at 13 55 29" src="https://user-images.githubusercontent.com/1260305/211313140-065d927b-e6cc-4cb5-86d5-8106091b9c47.png">

Reply:
I can add several possible prefixes for snippets to be able call them quickly and be able find them quickly too.

Reply:
Done. :)

Reply:
I've also put naming convention in this repo for snippets.